tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -518,6 +581,9 @@ void ordering_test_cases() {
         const std::filesystem::path p3{R"(a/b/d)"};
 
         spaceship_test<std::strong_ordering>(p1, p2, p3);
+        spaceship_test<std::strong_ordering>(p1, p2, p3);
+        spaceship_test<std::strong_ordering>(p1, p2, p3);

```

Comment:
Why are we repeating ourselves repeating ourselves?

Reply:
Ctrl CCC